setup.py

```diff
@@ -66,8 +67,11 @@
     ],
     "gpu": [
         "nvidia-ml-py3>=7.352.0",
-        "tensorflow-gpu>=1.15.0,<2",  # compiled with CUDA 10.0
-        "torch==1.2.0",  # last os-common version with CUDA 10.0 support
+        "tensorflow==2.6",  # compiled with CUDA 11.2, cudnn 8.1
```

Comment:
question about the pin, in TF<2.5 we are flagged, right? 

Second question, what would happen if we do TF>=2.6

Reply:
is tf 2.7 a possibility? does that resolve the numpy issue too, so we could move up to 1.20

Reply:
Here they say it needs numpy 1.19
https://github.com/tensorflow/tensorflow/blob/v2.7.0/tensorflow/tools/pip_package/setup.py
However, I just tried and it did install in a conda env along with numpy 1.20.
Also we would need to forget Python 3.6 with numpy 1.20.

Reply:
I tried TF 2.7 on virtual env too.

Reply:
@Leavingseason in case you have any comments on this.